app/bundles/ReportBundle/Event/ColumnCollectEvent.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\ReportBundle\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class ColumnCollectEvent extends Event
```

Comment:
I suggest to make all DTO objects final.
```suggestion
final class ColumnCollectEvent extends Event
```